src/controllers/trip.controller.js

```diff
@@ -56,7 +56,7 @@ class TripController {
       };
       const tripRequest = await tripService.CreateTripRequest(requestData);
       await NotificationService.sendNotification('trip_request_event', managerId, `New ${type} trip request.`, `${req.user.firstName} has requested a new ${type}`, tripRequest.id, 'http://localhost:3000/api/v1/notifications');
```

Comment:
@mujohn26 Kindly use the environment variable to point to a particular URL depending on the application environment